worker.go

```diff
@@ -11,15 +11,16 @@ import (
 
 // Worker is the doodad that does work.
 type Worker struct {
-	id        int
-	WorkChan  chan UDPMetric
-	QuitChan  chan struct{}
-	processed int64
-	imported  int64
-	mutex     *sync.Mutex
-	stats     *statsd.Client
-	logger    *logrus.Logger
-	wm        WorkerMetrics
+	id         int
+	WorkChan   chan UDPMetric
```

Comment:
I wonder if we shouldn't change the name of `WorkChan` now. That seems a bit tedious, but perhaps it's no longer clear what `work` is here to new code-readers. `PacketChan` perhaps?